client-sdk/ts-web/rt/src/types.ts

```diff
@@ -18,6 +18,43 @@ export interface CoreCallDataPublicKeyQueryResponse {
     public_key: KeyManagerSignedPublicKey;
 }
 
+/**
+ * Response to the RuntimeInfo query.
+ */
+export interface CoreRuntimeInfoQueryResponse {
```

Comment:
Are these TS bindings tested anywhere? I didn't find anything. Same for go.

For go, I tested the new method manually by creating a minimal runtime and its client as per https://docs.oasis.dev/oasis-sdk/guide/minimal-runtime. For TS, I was playing around trying to make `ts-web/rt/playground` work, but no luck in a reasonable amount of time. I can definitely work on this more but would appreciate some pointers.

Reply:
Yes.

For Go, see `tests/e2e`, for TS see `client-sdk/ts-web/rt/playground`.

Reply:
Thank you, I saw those. They are not extensive interface tests, they don't exhaustively cover all endpoints, and seem to focus more on core business logic.
Anyway, added my (trivial) tests to both of those folders.